docs/dev/event_time.md

```diff
@@ -213,10 +213,36 @@ arrive after the system's event time clock (as signaled by the watermarks) has a
 timestamp. See [Allowed Lateness]({{ site.baseurl }}/dev/stream/operators/windows.html#allowed-lateness) for more information on how to work
 with late elements in event time windows.
 
+## Idling sources
+
+Currently with pure event time watermarks generators, watermarks can not progress if there are no elements
```

Comment:
remote comma

Reply:
I have changed it to 
>Currently, with pure event time watermarks generators, watermarks can not progress if there are no elements to be processed. 

(added comma after currently)